src/components/Blog/Blog.react.js

```diff
@@ -206,6 +206,16 @@ class Blog extends Component {
     this.scrollToTop();
   };
 
+  showScrollToTop = () => {
+    let topButton = document.getElementById('topButton');
+    let y = window.scrollY;
```

Comment:
`y`?

Reply:
@akshatnitd window.scrollY gives the number of pixels the window is vertically scrolled from origin